### ISSUE TITLE
Modify PDF not showing additional content

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#9486]
**Describe the bug**
The attached PDF has an additional element that is not being shown when uploaded

**To Reproduce**
1. upload huh.pdf
2. Compare with huh.png -- it's missing a second 'AND' gate

**Expected behavior**
Both AND gates would be displayed

**Screenshots**
![huh](https://user-images.githubusercontent.com/290351/81499557-7859ee80-929a-11ea-8cda-43e8ff0250fe.png)
[huh.pdf](https://github.com/bigbluebutton/bigbluebutton/files/4605643/huh.pdf)

**Additional context**

**Comment by iSamof** [comment]
@ffdixon - I see the AND gate in both the PNG and also in the PDF ... I am viewing the PDG with MS Edge.  ... maybe it is just me!

**Comment by ffdixon** [comment]
Reported here: https://groups.google.com/d/msg/bigbluebutton-users/y_0LU-MrIY8/LGyAYddWAQAJ

**Comment by ffdixon** [comment]
Yes, when the PDF is uploaded to BigBlueButton, the second AND gate is missing in the presentation area display.

**Comment by sandrodegiorgi** [comment]
Fred asked me to to add here, how the pdf was modified. I asked my colleague, find original answer in german below. Trying to translate here now:

_He is working with word documents. He uses graphics and/or  parts from other pdf
documents (he converts these fragments to pdf with adobe-pdf) and inserts them
into his word document.

The "not-showing-up" elements are these inserted elements. Thing is: "sometimes
it works, sometimes not".

Workaround: He used the huh.pdf, snipped the missing part (e.g. the and-gate),
re-inserted it into the word document, converted the word to pdf, 
and everything works fine then._

---[original german reply]---
_Das sind Word-Dokumente mit eingefügten Grafiken oder Ausschnitten aus
PDF-Dokumenten, die ich über Adobe-PDf in PDF-Dokumente umgewandelt habe.

Die Elemente, die dann nach dem Hochladen verschwinden sind eingefügte
Grafiken oder eingefügte Ausschnitte aus anderen PDF-Dokumenten, z.B.
aus alten Abi-Prüfungen, die als PDF vorliegen.
Aber das Ganze klappt nur manchmal eben nicht.

Habe ich dann aber aus dem von mir, aus dem Word-Dokument, generierten
PDF den gleichen Ausschnitt nochmals ausgeschnitten und diesen dann
wieder in Word eingefügt und dann erneut in PDF konvertiert, dann war
alles gut._

Greetings,
Sandro
